test/tests/functional/pbs_systemd.py

```diff
@@ -110,3 +112,53 @@ def test_systemd(self):
         rv = self.stop_using_systemd()
         self.assertTrue(rv)
         rv = self.start_using_systemd()
+
+    @skipOnShasta
+    def test_missing_daemon(self):
+        """
+        Test whether missing daemons starts without re-starting other daemons
+        """
+        self.hostname = self.server.hostname
+        self.shutdown_all()
+        rv = self.start_using_systemd()
+        self.assertTrue(rv)
+        # Mom
+        self.mom.signal("-KILL")
+        if self.mom.isUp(max_attempts=10):
+            self.fail("MoM is still running")
+        cmd = "systemctl reload pbs"
+        self.du.run_cmd(self.hostname, cmd, True)
+        if self.mom.isUp(max_attempts=10):
+            self.logger.info("MoM started and running")
+        else:
+            self.fail("MoM not started")
+        # Sched
+        self.scheduler.signal("-KILL")
+        if self.scheduler.isUp(max_attempts=10):
+            self.fail("Sched is still running")
+        cmd = "systemctl reload pbs"
+        self.du.run_cmd(self.hostname, cmd, True)
```

Comment:
You don't need to keep reassigning the same "systemctl reload pbs" to cmd...

Reply:
made the changes